browser/net/decentralized_dns_network_delegate_helper.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_NET_DECENTRALIZED_DNS_NETWORK_DELEGATE_HELPER_H_
+#define BRAVE_BROWSER_NET_DECENTRALIZED_DNS_NETWORK_DELEGATE_HELPER_H_
+
+#include <memory>
+#include <string>
+
+#include "brave/browser/net/url_context.h"
+#include "net/base/completion_once_callback.h"
+
+namespace decentralized_dns {
+
+int OnBeforeURLRequest_DecentralizedDnsPreRedirectWork(
```

Comment:
Please add a comment describing what this is for

Reply:
1cdb4f081decc67ac46ac53ed9bee83fcf3ebb34